pkgs/development/compilers/sdcc/zsdcc.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, sdcc, fetchFromGitHub, ... }:
```

Comment:
Please remove ```...``` and list each needed input explicitly.

Reply:
will i not lost the interest of derivate sdcc doing that ?

Reply:
I don't understand that question.